ml_tools/utils/plotting.py

```diff
@@ -224,3 +226,36 @@ def print_metrics(models:       Dict[str, PredictionStrategy],
         avg = np.mean(diff)
         std = np.std(diff)
         print(fmtstr.format(label + ' :', avg, std, rms, maxdiff, dt/len(state_series)))
+
+
+def plot_corr_matrix(input_features:  List[str],
+                     state_series:    List[StateSeries],
+                     state_index:     int = -1,
+                     fig_name:        str = 'corr_matrix') -> None:
+    """ Function for plotting the correlation matrix of a
```

Comment:
the cliffhanger is killing me...

Also leave a blank line between this and `Parameters`

Reply:
blllaaaarggg.  Fixed.